src/fixup-svg-string.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Fixup svg string prior to parsing.
+ * @param {!string} svgString String of the svg to fix.
+ * @returns {!string} fixed svg that should be parseable.
+ */
+module.exports = function (svgString) {
+    // Add root svg namespace if it does not exist.
+    const svgAttrs = svgString.match(/<svg [^>]*>/);
+    if (svgAttrs && svgAttrs[0].indexOf('xmlns=') === -1) {
+        svgString = svgString.replace('<svg ', '<svg xmlns="http://www.w3.org/2000/svg" ');
+    }
+
+    // There are some SVGs from Illustrator that use undeclared entities.
+    // Just replace those entities with fake namespace references to prevent
+    // DOMParser from crashing
+    if (svgAttrs && svgAttrs[0].indexOf('&ns_') !== -1 && svgString.indexOf('<!DOCTYPE') === -1) {
```

Comment:
Would it make more sense to do this only if '&ns_' appears anywhere else in the svgString?

Reply:
All the svgs that I saw seemed to use them at the top level svg, and i wanted to avoid any issues with using &ns in svg text or something